content.test.ts

```diff
@@ -268,6 +280,63 @@ test("test streaming content", { timeout: 180000 }, async () => {
   expect(refinedContent.title).toBe(title);
   expect(refinedContent.version).toBe(1);
   expect(refinedContent.commands.length).toBe(2);
+  expect(refinedContent.status).toBe("DRAFT");
+  expect(refinedContent.publishedVersion).toBe(undefined);
 
   await catalog.delete();
 });
+
+test(`test content status and publishing`, { timeout: 180000 }, async () => {
+  testClient.configureOrg({
```

Comment:
Note, we configure this org/cortex/catalog so much I wonder if it would be worth refactoring into a util? We should only really be configuring the org once at the start of tests.

Reply:
Good idea, I'll move it to a beforeAll